### PR TITLE
(Milestone 1) To-Do-List: List-Structure

### PR DESCRIPTION
**Pull request by DebbyMiressa**
With the 'List-Structure' Branch, the following features were implemented.
1) Setup Project with Webpack where the web pack is useful to bundle JS files together for efficient execution.
2) Add To-Do List using an unordered list to index.html
3) Add an array of objects to index.js whereby we add an array of list items (tasks) and populate an HTML unordered list by iterating through this array.
4) Add styling to src/style.css and also incorporate web pack style/CSS loader to webpack.config.js

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    this.todolist = [
+      {
+        description: 'Wash the dishes',
+        completed,
+        index: 1,
+      },
+      {
+        description: 'Complete to-do list project',
+        completed,
+        index: 2,
```
Comment (julie-ify) [inline src/index.js:20]: - [ ] The `completed` attribute should have a boolean value (`true` or `false`). Kindly do that.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,135 @@
+<div id="top"></div>
+
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
```
Comment (julie-ify) [inline README.md:3]: - [ ] Kindly edit your pull request title and description to be more descriptive of the feature(s) you are trying to merge. Merely mentioning the branch name and base branch is not very elaborate.

  pr-descr

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @DebbyMiressa,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (FrederickMih) [inline README.md:3]: - [**Optional**] Let's always ensure that the PR title reminds us of the particular feature we are implementing for that project. The project you are building is the **To-Do-List** and the specific aspect you are working on this project right now is `list structure`. You may simply name it `To-Do-List: list structure`

**Review by FrederickMih — APPROVED** [review]
# Approved :tada: 
 Hi @DebbyMiressa , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (DebbyMiressa) [reply README.md:3]: Thanks, I will fix it accordingly. 👍

**Pull request merged**